build_oblivion_remastered_german_voices.bat

```diff
@@ -218,9 +210,9 @@ if !AMOUNT_BNK_BEFORE! lss 133000 (
 if %EXECUTE_MP3_DIFF_SCRIPT% == "true" (
     .\busybox\busybox.exe bash scripts\check-missing-wems.sh
 )
-
+echo Building the Mod PAK file...
 :: Final step. Build the mod PAK file
-cmd /c .\voxmeld\create-pak.exe .\german-voices-oblivion-remastered-voxmeld_v%VERSION_NUMBER%_P\
+cmd /c .\repak\repak.exe pack --prefix "OblivionRemastered" --version V11 .\german-voices-oblivion-remastered-voxmeld_v%VERSION_NUMBER%_P\ "%RESULT_FOLDER_PAK%\german-voices-oblivion-remastered-voxmeld_v%VERSION_NUMBER%_P.pak"

```

Comment:
```suggestion
cmd /c .\repak\repak.exe pack -m "../../../OblivionRemastered/" --version V11 .\german-voices-oblivion-remastered-voxmeld_v%VERSION_NUMBER%_P\ "%RESULT_FOLDER_PAK%\german-voices-oblivion-remastered-voxmeld_v%VERSION_NUMBER%_P.pak"

```

Shouldn't it be possible to use the official [repak](https://github.com/trumank/repak) and just configure the mount point here?
It's working great on my side.